flyteadmin_config.yaml

```diff
@@ -159,3 +159,16 @@ cluster_resources:
       valueFrom:
         env: SHELL
   refresh: 3s
+qualityOfService:
```

Comment:
Not a fan of this name - I just don't like the connotation of a low quality of service.  Is it too late to change this?

Reply:
it's not used so technically i can go make a breaking change :) any suggestions instead? basic/baseline?

Reply:
actually we've had bad luck with supposedly no-op breaking changes so I would rather not (this is checked into flyteidl, unfortunately)